src/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+## crypto plugins
+
+set(crypto_plugin_dir ${CMAKE_INSTALL_PKGLIBDIR}/crypto)
+set(crypto_plugins_deps "")
+
+if(HAVE_INTEL AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE))
+  add_subdirectory(isa-l)
+  list(APPEND crypto_plugins_deps ceph_crypto_isal)
+endif(HAVE_INTEL AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE))
+
+if(WITH_QAT)
+  add_subdirectory(qat)
```

Comment:
is this really necessary?

Reply:
I assumed not everyone would like the build yet another library if they didn't really need to.  Ohh there is also the need to have the kernel headers to be able to build QAT which is not part of the install-deps.sh (which i assumed forcing everyone to install just cause QAT is here) was not something that would be good.

Reply:
@ganeshmaharaj i meant to add the include directory globally is not necessary.